server/eclipse-project/src/main/java/us/freeandfair/corla/csv/ColoradoBallotManifestParser.java

```diff
@@ -126,6 +137,20 @@ public ColoradoBallotManifestParser(final String the_string,
   }
   
   /**
+   * Checks to see if the set of parsed manifests needs flushing, and does so 
+   * if necessary.
+   */
+  private void checkForFlush() {
+    if (my_parsed_manifests.size() % BATCH_SIZE == 0) {
```

Comment:
It would not. It doesn't matter whether it flushes for any multiple of batch size, because we don't care - the transaction will be committed after that. This is just to make sure we don't keep too much in memory at once.